lib/mwoffliner.lib.js

```diff
@@ -2615,13 +2545,7 @@ function execute(argv) {
 
       var html = '<ul>\n';
       Object.keys(articleIds).sort().map(function(articleId) {
-        html =
-          html +
-          '<li><a href="' +
-          env.getArticleBase(articleId, true) +
-          '"\>' +
```

Comment:
Worth checking separately (from this PR) why \> vs >?

Reply:
I don't think I understand what you mean by `Worth checking separately (from this PR) why > vs >?`

But yes, I fixed the merge conflicts. I was conflicting against my own changes. All fixed up.

Reply:
github ate up my characters. "\\>" vs ">" . i.e. why s the a-tag closing with a "\\>"

Reply:
That line isn't written by me originally. I'm not sure. Escaping the ">"?